code/modules/research/designs/limbgrower_designs.dm

```diff
@@ -266,8 +250,8 @@
 	build_path = /obj/item/disk/design_disk/limbs/felinid
 
 /obj/item/disk/design_disk/limbs/lizard
-	name = "Lizard Limb Design Disk"
-	limb_designs = list(/datum/design/lizard_tail, /datum/design/lizard_tongue, /datum/design/digi_leftleg, /datum/design/digi_rightleg)
+	name = "Lizard Organ Design Disk"
+	limb_designs = list(/datum/design/lizard_tail, /datum/design/lizard_tongue)
```

Comment:
All of them are called "limb design disks", regardless if they make any limbs or not. If you're gonna change this one might as well make the rest consistent? :)